### PR TITLE
Refresh of the rule metadata

### PR DESCRIPTION
**Pull request by ivandalbosco**

### CODE REVIEW

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S104.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Files should not have too many lines",
+  "title": "Files should not have too many lines of code",
```
Comment (vilchik-elena) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S104.json]: @ivandalbosco We should not include this change before https://jira.sonarsource.com/browse/SONARJS-876 is done

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S138.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Functions should not have too many lines",
+  "title": "Functions should not have too many lines of code",
```
Comment (vilchik-elena) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S138.json]: @ivandalbosco We should not include this change before https://jira.sonarsource.com/browse/SONARJS-876 is done

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S104.json (hunk 1 above)
Comment (ivandalbosco) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S104.json]: @vilchik-elena OK

File: javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S138.json (hunk 2 above)
Comment (ivandalbosco) [inline javascript-checks/src/main/resources/org/sonar/l10n/javascript/rules/javascript/S138.json]: @vilchik-elena OK

**Pull request merged**